### DISCUSSION TITLE
Fluid wallpaper on mac

### DISCUSSION THREAD

**Discussion by haydxxn** [rocksdanister/lively#1885] [General]
I really like Fluid wallpaper but I'm using Mac. So are there any ways to set just Fluid wallpaper on Mac using its Javascript code in the Fluid folder or something like that?

Btw, hope u develop the Mac version when u free! ty

**Comment by rocksdanister** [comment]
There is this app:
https://github.com/sindresorhus/Plash

that can load webpages:
https://paveldogreat.github.io/WebGL-Fluid-Simulation/

**Reply by haydxxn** [reply]
I tried it! it does not support multiple desktops! so I hope your app will be available on Mac soon! ty
